js/tab.js

```diff
@@ -14,7 +14,7 @@
   // ====================
 
   var Tab = function (element) {
-    this.element = $(element)
+    this.$element = $(element)
```

Comment:
I agree this is stylistically nicer, but it has the potential to break folks' code if they're extending the Tab plugin...